install/files/deb-packages_requirements.txt

```diff
@@ -1,22 +1,14 @@
 libssl-dev
```

Comment:
As we do not use the docker images anymore do we still need this file ??

Reply:
We still use it for Debian, right?

Reply:
We do not use the `setup.py` ?

Reply:
It's system library. Not available from pip.

Reply:
I  confirm, this system package is used by some pip package like ansible